src/main.cpp

```diff
@@ -532,6 +533,22 @@ int main ( int argc, char** argv )
         }
 
 
+        // Jack server name ----------------------------------------------------
```

Comment:
Could we only add this logic if we have jack support at all (`#ifdef WITH_SOUND`, it seems. #1408)?

(I know that the existing code for jack options does not lead by good example in this regard, we should try to be better nevertheless :))

Reply:
I updated my commit